src/ui/public/styles/variables/colors.less

```diff
@@ -0,0 +1,70 @@
+//== Colors
+//
+//## Gray and brand colors for use across Bootstrap.
+
+@white:                                 #ffffff;
+@black:                                 #000000;
+
+@gray2:                                 #222222;
+@gray3:                                 #333333;
+@gray4:                                 #444444;
+@gray5:                                 #555555;
+@gray6:                                 #666666;
+@gray7:                                 #777777;
+@gray8:                                 #888888;
+@gray9:                                 #999999;
+@gray10:                                #aaaaaa;
+@gray11:                                #bbbbbb;
+@gray12:                                #cccccc;
+@gray13:                                #dddddd;
+@gray14:                                #eeeeee;
+
+@gray-base:                             @black;
+@gray-darker:                           @gray2;
+@gray-dark:                             #7b8a8b;
+@gray:                                  #95a5a6;
+@gray-light:                            #b4bcc2;
+@gray-lighter:                          #ecf0f1;
+
+@brand-primary:                         @gray4;
```

Comment:
I don't understand the distinction of what got moved into this colors.less file. The above colors make sense because they are variables that hold color values. Even the 'brand' variables make sense in here because it is assigning a color to a named variable. The rest of the variables that were moved into this file don't make sense here.